src/components/interfaces/MOBILE_API.xml

```diff
@@ -5369,65 +5454,149 @@
 
   <function name="OnEncodedSyncPData" functionID="OnEncodedSyncPDataID" messagetype="notification" >
     <description>
-    	Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.
-    	Legacy / v1 Protocol implementation; responds to EncodedSyncPData.
-    	*** DEPRECATED ***
+      Callback including encoded data of any SyncP packets that SYNC needs to send back to the mobile device.

```

Comment:
@LuxoftAKutsan formatting should be done in this file. You could view actual changes using ignore whitespace flag. See this commit using this link https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215?w=1